### DISCUSSION TITLE
Bitbucket Cloud: Should the renovate/stability-days build hyperlink to the stabilityDays configuration option?

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#21008] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Current behavior when using the [stabilityDays](https://docs.renovatebot.com/configuration-options/#stabilitydays) configuration option with Bitbucket Cloud is the following artificial build

![Screenshot 2023-03-17 at 10 54 43 AM](https://user-images.githubusercontent.com/386277/225940872-1c291048-25d2-4b30-9334-e0830fca94f4.png)

When clicking the build name, it launches https://docs.renovatebot.com/

Would there be interest in have this link to https://docs.renovatebot.com/configuration-options/#stabilitydays , or something else more specific?

**Comment by rarkins** [comment]
Yes, it could be worth having a dedicated page in fact, considering that we link to it from so many PRs

**Reply by setchy** [reply]
Raised as: https://github.com/renovatebot/renovate/issues/21013

**Comment by setchy** [comment]
Closing discussion in favor of this more rich question - https://github.com/renovatebot/renovate/discussions/14948
